### PR TITLE
Add basic documentation

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -57,7 +57,10 @@ before_script:
   - "sh -e /etc/init.d/xvfb start"
   - sleep 3
 script:
+  - flake8 setup.py
```
Comment (stefanoborini) [inline .travis.yml]: ok, I see the problem. You want to skip flake traversing the doc directory. I would restore it as is, and use the .flake8 configuration file, like this 

```
 $ more .flake8 
[flake8]
exclude = build/*,venv/*,doc/source/*,tests/utils.py,selenium_tests/*
```

File: doc/source/ui_doc.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Using the UI, you will be able to set the parameters of the computation (number of
+iterations, time step, boundary conditions...).
+
+You must specify input files for OpenFOAM and Liggghts. You can find
+examples of input files in the simphony_ui/tests/fixtures directory.
+
+Run the calculation
+-------------------
+
+Once input parameters correctly set, you'll be able to run the calculation with
```
Comment (stefanoborini) [inline doc/source/ui_doc.rst]: are correctly set

File: doc/source/ui_doc.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+the "Run" button at the bottom of the dialog. A progress bar dialog should appears.
```
Comment (stefanoborini) [inline doc/source/ui_doc.rst]: should appear

File: doc/source/conf.py [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+
+
+sphinx.environment.BuildEnvironment.warn_node = _warn_node
+
+def mock_modules():
+    import sys
+
+    from mock import MagicMock
+
+    try:
```
Comment (stefanoborini) [inline doc/source/conf.py]: This one and the other try are technically not needed. I would comment one out as an example (comment it as such) and remove the other

**Review by stefanoborini — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (martinRenou) [reply .travis.yml]: Oh ok thanks I didn't know how to do it :)

File: doc/source/conf.py [hunk 5]
```diff
@@ -27,17 +27,12 @@ def mock_modules():
 
     from mock import MagicMock
 
-    try:
+    """try:
```
Comment (stefanoborini) [inline doc/source/conf.py]: Nah... Use # and also write it that it's an example.

**Review by stefanoborini — CHANGES_REQUESTED** [review]

**Review by stefanoborini — APPROVED** [review]
